bin/verify-docs.sh

```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
-
-set -ev
-
-# Make sure we use the correct repo root dir
-DIR=`dirname $0`
-REPO_ROOT=$DIR/..
-pushd $REPO_ROOT >/dev/null
-rm -rf sandbox/loopback.io/
-# Shadow clone is faster
-git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
-lerna bootstrap --scope loopback.io-workflow-scripts
```

Comment:
Thanks for the fix @raymondfeng ! A simpler fix would be to just add `@loopback/build` in the scope so it's bootstrapped. The following worked for me. 

`lerna bootstrap --scope loopback.io-workflow-scripts --scope @loopback/build --scope @loopback/docs`

Reply:
I would rather have all the steps in the shell script itself. `@loopback/build` has a lot of dependencies and it's time consuming to bootstrap it. Please note the PR also adds a lot of comments in the script.

Reply:
+1 for having them in the shell script if that is the case.

Reply:
Fair enough! And yay for the comments!